plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -0,0 +1,1072 @@
+                      ctx->log_group, 0)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "\",\"logStreamName\":\"", 19)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      stream->name, 0)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "\",", 2)) {
+        goto error;
+    }
+
+    if (stream->sequence_token) {
+        if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                          "\"sequenceToken\":\"", 17)) {
+            goto error;
+        }
+
+        if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                          stream->sequence_token, 0)) {
+            goto error;
+        }
+
+        if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                          "\",", 2)) {
+            goto error;
+        }
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "\"logEvents\":[", 13)) {
+        goto error;
+    }
+
+    return 0;
+
+error:
+    return -1;
+}
+
+/*
+ * Writes a log event to the output buffer
+ */
+static int write_event(struct flb_cloudwatch *ctx, struct cw_flush *buf,
+                       struct event *event, int *offset)
+{
+    char ts[50];
+
+    if (!sprintf(ts, "%llu", event->timestamp)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "{\"timestamp\":", 13)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      ts, 0)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      ",\"message\":\"", 12)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      event->json, event->len)) {
+        goto error;
+    }
+
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "\"}", 2)) {
+        goto error;
+    }
+
+    return 0;
+
+error:
+    return -1;
+}
+
+/* Terminates a PutLogEvents payload */
+static int end_put_payload(struct flb_cloudwatch *ctx, struct cw_flush *buf,
+                           int *offset)
+{
+    if (!try_to_write(buf->out_buf, offset, buf->out_buf_size,
+                      "]}", 2)) {
+        return -1;
+    }
+    buf->out_buf[*offset] = '\0';
+
+    return 0;
+}
+
+static unsigned long long stream_time_span(struct log_stream *stream,
+                                           struct event *event)
+{
+    if (stream->oldest_event == 0 || stream->newest_event == 0) {
+        return 0;
+    }
+
+    if (stream->oldest_event > event->timestamp) {
+        return stream->newest_event - event->timestamp;
+    } else if (stream->newest_event < event->timestamp) {
```

Comment:
```else if``` in the next line:

https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions